vignettes/teal-reporter-blocks-overview.Rmd

```diff
@@ -0,0 +1,120 @@
+---
+title: "Comprehensive Overview of Content Blocks in `teal.reporter`"
```

Comment:
When making pages filenames should be the same as titles. If they are different then it is more difficult to navigate in the project when developing.

Reply:
done [d70eefe](https://github.com/insightsengineering/teal.reporter/pull/272/commits/d70eefe8976abb78d0ef76a14ba3637e8dd5efcf)